docs/install_microk8s.md

```diff
@@ -0,0 +1,131 @@
+## 1. Overview
+Kubeflow is a novel open source tool for Machine Learning workflow orchestration on kubernetes. It has great powers, but deploying it may not be so easy, depending on how and where you deploy your kubernetes. 
+This tutorial will show you an easy way to deploy kubeflow using microk8s, a lightweight version of kubernetes, in a few simple steps.
+
+### What you'll learn
+
+* How to deploy microk8s on ubuntu
+* How to deploy kubeflow using microk8s
+* How to access your kubeflow dashboard 
+
+### What you'll need
+* Desktop or Virtual Machine with Ubuntu 16.04 or above
+* A minimum of 4 CPU, 14GB RAM, 50GB Disk (recommended 8 CPU, 32GB RAM, 60GB Disk)
+* Hyper V, if using multipass on a Windows machine (not available on Windows 10 Home)
+* Some basic command-line knowledge
+
+## 2. Open ubuntu terminal
+
+All we need to do here is access the terminal of the machine we want to deploy kubeflow to. This is fairly straightforward.
+
+### Option 1: on ubuntu machine
+
+1. Open terminal (CTRL + Alt + t )
+
+### Option 2:  accessing remote ubuntu virtual machine
+
+1. Spin up a virtual machine with Ubuntu (on a server or cloud)
+
+2. Access the machine terminal using `ssh`:
+
+ `$ ssh ubuntu@<machine-ip-address>`
+
+### Option 3: on Windows or MacOS 
+
+1. Download multipass from *multipass.run* 
+
+2. Launch Multipass instance with enough requirements:
+
+`$ multipass launch --name kubeflow --mem 16G --disk 50G --cpus 4`
+
+3. Access Multipass instance:
+
+`$ multipass shell kubeflow`
+
+
+## 3. Install Microk8s 
+
+In this step you will install MicroK8s, a minimal, lightweight Kubernetes you can run and use on practically any machine. It can be installed with a snap:
+
+
+
+1.  Install Microk8s: 
+
+`$ sudo snap install microk8s --classic`
+
+2. Verify installation:
+
+`$ sudo microk8s status --wait-ready`
+
+
+## 4. Join admin group
+
+MicroK8s creates a group to enable seamless usage of commands which require admin privilege. 
+
+
+1. To add your current user to the group and gain access to the  `.kube`  caching directory, run the following two commands:
+
+` sudo usermod -a -G microk8s $USER `
```

Comment:
We could probably just join these two commands into a single code block, e.g. like this:

```
    sudo usermod -a -G microk8s $USER
    sudo chown -f -R $USER ~/.kube
```
It can be done with either 3 backticks above and below the lines, or by indenting each line by four spaces, as shown.